src/dftracer/core/dftracer_main.cpp

```diff
@@ -3,6 +3,11 @@
 //
 #include <dftracer/core/dftracer_main.h>
 #include <dftracer/finstrument/functions.h>
+#include <dftracer/function/hip/intercept.h>
+
+// TODO: Figure out why I have to manually enable it
+// #define DFTRACER_HIP_TRACING_ENABLE 1
+
```

Comment:
Is this still a concern? You  need to use the dftracer_config.hpp trick I shared before.